src/edi_energy_scraper/__init__.py

```diff
@@ -301,6 +303,27 @@ async def _download(
             raise
         return file_path
 
+    @staticmethod
+    def get_edifact_format(path: Path) -> Tuple[EdifactFormatVersion, List[Optional[EdifactFormat]]]:
```

Comment:
vllt edifact_format**s** (plural) and version? wobei der return-ype hint ja schon viel verrät.

Reply:
tjaja good old naming
vielleicht so `get_edifact_format_and_version`